data_pipeline/generic_web_api/transform_data.py

```diff
@@ -84,6 +88,27 @@ def standardize_record_keys(record_object):
         return new_list
 
 
+def _is_valid_timestamp_string(timestamp_str: str) -> bool:
+    try:
+        if dateparser.parse(timestamp_str):
+            return True
+    except BaseException:
+        pass
+    return False
+
+
+def _get_valid_timestamp_string_or_none(
```

Comment:
Just wondered why do we need to have `_` beginning of this function ? And maybe in previous (`_is_valid_timestamp_string`) one too?

Reply:
We discussed this in our data catch-up. There are no strict rules. But my thinking was those methods, within a utility module, are there to make another function simpler. But I wouldn't want it to be used elsewhere. (Especially with the questionable use of `dateparser` which is a bit too lax)